### ISSUE TITLE
Jobs/Events not rendering in dev mode

### ISSUE THREAD

**Issue by eupiteco** [opensourcedesign/opensourcedesign.github.io#263]
I cloned and installed the repo and got no way to render the the events and jobs pages w/ default configs. My solution was to move `/_posts/jobs` to `/_jobs`, `/_posts/events` to `/_events` and include a permalink tag to each index file.

Am I doing it right? Is there something I'm missig?

**Comment by RDIL** [comment]
Did you clone the subrepos?

**Comment by eupiteco** [comment]
Yeah, I ran the install script accodring to the documentation and it clones the subrepos but the config_dev.yaml can't seem to findo the repos where they are cloned by default.

**Comment by RDIL** [comment]
In that case, I'll mark this as pending further investigation.

**Comment by im-adithya** [comment]
Weren't the `_events` and `_jobs` folders outside `_posts` folder by default?
Didn't face this issue (just cloned the repo and everything worked fine)

**Comment by Erioldoesdesign** [comment]
Hey there @eupiteco can you please re-check this and consider what @im-adithya has said - re. the jobs and events are outside the posts folder by design and close this is this is what the problem is

**Comment by Ian2020** [comment]
Some things have changed since this issue was posted but I have the same problem. Grabbing the code as the current README suggests:

```
git clone https://github.com/opensourcedesign/opensourcedesign.github.io.git
cd opensourcedesign.github.io/
git clone https://github.com/opensourcedesign/jobs.git
```

...we end up with the jobs repo in dir `jobs`. How is this combined/overlaid with the main site properly? I see in `_config-dev.yml` we have this:

```
# Sub-Repos
include: ["_jobs", "_organization", "_patterns", "_talks"]
```

...but there is no `_jobs` to include(?) When I do serve the site:

```
bundler exec jekyll serve --watch --config _config.yml,_config-dev.yml
```

...the jobs posts appear at `jobs/jobs`:

http://127.0.0.1:4000/jobs/jobs/2025-04-17-create-explanatory-hero-graphic-for-website.html

...however they are using 5 year old HTML from the main repo i.e. https://github.com/opensourcedesign/opensourcedesign.github.io/blob/master/_layouts/jobs.html

I must be missing something?

**Comment by AnXh3L0** [comment]
New website is using Hugo, and the repo has become a monorepo, bundling the job posts inside it, closing issue because it's not applicable anymore.
